### PR TITLE
chore: Angularjs section for other tech block

### PR DESCRIPTION
**Pull request by DonNicoJs**
Basic AngularJs block for Coming from other technology
To see if the style and content is okay before proceeding with Polymer

### CODE REVIEW

File: docs/guide/learning/how-to-learn-vue.md [hunk 1]
```diff
@@ -195,6 +195,29 @@ In this section, we will go over the most popular technologies and the struggles
 
 ### AngularJS (Angular 1.x)
 
+**Similarities**
+
+- Templates are pure HTML
```
Comment (dobromir-hristov) [inline docs/guide/learning/how-to-learn-vue.md]: I would rephrase that they both use HTML based templating systems. The directives and what knot are technically not "pure" hehe, you get my idea.

File: docs/guide/learning/how-to-learn-vue.md [hunk 2]
```diff
@@ -195,6 +195,29 @@ In this section, we will go over the most popular technologies and the struggles
...
+- Directives
+- Filters
+- Component approach (AngularJS from 1.5.x)
+
+**Differences**
+
+The main difference between VueJs and AngularJs is how reactivity is implemented, Angular has dirty checking phases where a check is done in regards of what model is out of sync with the DOM and updates are performed. Possible issues in converting an AngularJS codebase to a VueJS one is to properly adhere to best practices. (IE: AngularJs encourage an extensive use of filters while in VueJs is preferrable to use computed properites)
```
Comment (dobromir-hristov) [inline docs/guide/learning/how-to-learn-vue.md]: I would follow the bullet pattern from above (to keep consistency, but then for each bullet do the summary. Much easier to scan.

File: docs/guide/learning/how-to-learn-vue.md [hunk 3]
```diff
@@ -195,6 +195,29 @@ In this section, we will go over the most popular technologies and the struggles
...
+
+ - Dirty checking of AngularJs can cause loops and be extremely slow
+ - Unit testing  AngularJs components can be tricky
+ - VueJs has a much smaller footprint
+ - VueJs is less verbose
+ - AngularJs community packages are often abandoned ( or only maintained for Angular 2x, 3x ... )
+
+**Links**
+
+[VueJs Docs](https://vuejs.org/v2/guide/comparison.html#AngularJS-Angular-1)
```
Comment (dobromir-hristov) [inline docs/guide/learning/how-to-learn-vue.md]: Could you use the link component? 
```
<useful-links>
<useful-links-section title="Tutorials">

 [VueJs Docs](https://vuejs.org/v2/guide/comparison.html#AngularJS-Angular-1) 

</useful-links-section>
</useful-links

File: docs/guide/learning/how-to-learn-vue.md [hunk 4]
```diff
@@ -195,6 +195,29 @@ In this section, we will go over the most popular technologies and the struggles
...
+- Component approach (AngularJS from 1.5.x)
+
+**Differences**
+
+The main difference between VueJs and AngularJs is how reactivity is implemented, Angular has dirty checking phases where a check is done in regards of what model is out of sync with the DOM and updates are performed. Possible issues in converting an AngularJS codebase to a VueJS one is to properly adhere to best practices. (IE: AngularJs encourage an extensive use of filters while in VueJs is preferrable to use computed properites)
+
+**VueJs Benefits**
+
+ - Dirty checking of AngularJs can cause loops and be extremely slow
+ - Unit testing  AngularJs components can be tricky
```
Comment (dobromir-hristov) [inline docs/guide/learning/how-to-learn-vue.md]: Lets rephrase these from a Vue point of ... view? `Unit testing Vue components is easier than AngularJs`?

File: docs/guide/learning/how-to-learn-vue.md [hunk 5]
```diff
@@ -195,6 +195,29 @@ In this section, we will go over the most popular technologies and the struggles
...
+ - VueJs has a much smaller footprint
+ - VueJs is less verbose
+ - AngularJs community packages are often abandoned ( or only maintained for Angular 2x, 3x ... )
```
Comment (dobromir-hristov) [inline docs/guide/learning/how-to-learn-vue.md]: Vue has a very active and evolving community, compared to AngularJs, bla bla

File: docs/guide/learning/how-to-learn-vue.md [hunk 6]
```diff
@@ -195,6 +195,29 @@ In this section, we will go over the most popular technologies and the struggles
...
+
+**Links**
```
Comment (dobromir-hristov) [inline docs/guide/learning/how-to-learn-vue.md]: We can skip the Links text if we use the component

File: docs/guide/learning/how-to-learn-vue.md (hunk 1 above)
Comment (DonNicoJs) [reply docs/guide/learning/how-to-learn-vue.md]: Done!

File: docs/guide/learning/how-to-learn-vue.md (hunk 4 above)
Comment (DonNicoJs) [reply docs/guide/learning/how-to-learn-vue.md]: Done!

File: docs/guide/learning/how-to-learn-vue.md (hunk 6 above)
Comment (DonNicoJs) [reply docs/guide/learning/how-to-learn-vue.md]: Done!

File: docs/guide/learning/how-to-learn-vue.md (hunk 5 above)
Comment (DonNicoJs) [reply docs/guide/learning/how-to-learn-vue.md]: Done!

File: docs/guide/learning/how-to-learn-vue.md (hunk 2 above)
Comment (DonNicoJs) [reply docs/guide/learning/how-to-learn-vue.md]: Done!

File: docs/guide/learning/how-to-learn-vue.md [hunk 7]
```diff
@@ -195,6 +195,34 @@ In this section, we will go over the most popular technologies and the struggles
...
+
+- Both use a templating system based on HTML
+- Directives
+- Filters
+- Component approach (AngularJS from 1.5.x)
+
+**Differences**
+
+- Reactivity is implemented differently, Angular has dirty checking phases where a check is done in regards of what model is out of sync with the DOM and updates are performed. 
+- Possible issues in converting an AngularJS codebase to a VueJS one is to properly adhere to best practices. (IE: AngularJs encourage an extensive use of filters while in VueJs is preferrable to use computed properites)
```
Comment (dobromir-hristov) [inline docs/guide/learning/how-to-learn-vue.md]:
```suggestion
- Possible issues when converting an AngularJS codebase to Vue - Best practices differ, AngularJs encourages extensive use of filters while in Vue computed properties are preferred).
```

File: docs/guide/learning/how-to-learn-vue.md [hunk 8]
```diff
@@ -195,6 +195,34 @@ In this section, we will go over the most popular technologies and the struggles
...
+**Similarities**
+
+- Both use a templating system based on HTML
+- Directives
+- Filters
+- Component approach (AngularJS from 1.5.x)
+
+**Differences**
+
+- Reactivity is implemented differently, Angular has dirty checking phases where a check is done in regards of what model is out of sync with the DOM and updates are performed.
```
Comment (dobromir-hristov) [inline docs/guide/learning/how-to-learn-vue.md]:
```suggestion
- Reactivity implementation - Angular has dirty checking phases, where a check is done in regards of what model is out of sync with the DOM and updates are performed.
```

File: docs/guide/learning/how-to-learn-vue.md [hunk 9]
```diff
@@ -195,6 +195,34 @@ In this section, we will go over the most popular technologies and the struggles
...
+- Directives
+- Filters
+- Component approach (AngularJS from 1.5.x)
+
+**Differences**
+
+- Reactivity is implemented differently, Angular has dirty checking phases where a check is done in regards of what model is out of sync with the DOM and updates are performed. 
+- Possible issues in converting an AngularJS codebase to a VueJS one is to properly adhere to best practices. (IE: AngularJs encourage an extensive use of filters while in VueJs is preferrable to use computed properites)
+
+**VueJs Benefits**
```
Comment (dobromir-hristov) [inline docs/guide/learning/how-to-learn-vue.md]:
```suggestion
 **Why Switch?** 
```

File: docs/guide/learning/how-to-learn-vue.md [hunk 10]
```diff
@@ -195,6 +195,34 @@ In this section, we will go over the most popular technologies and the struggles
...
+
+ - Dirty checking of AngularJs can cause loops and be extremely slow
+ - Unit test VueJs component is easier than AngularJs
```
Comment (dobromir-hristov) [inline docs/guide/learning/how-to-learn-vue.md]:
```suggestion
 - Unit testin Vue components is easier
```

File: docs/guide/learning/how-to-learn-vue.md [hunk 11]
```diff
@@ -195,6 +195,34 @@ In this section, we will go over the most popular technologies and the struggles
...
+ - VueJs has a much smaller footprint
```
Comment (dobromir-hristov) [inline docs/guide/learning/how-to-learn-vue.md]:
```suggestion
 - Vue has a much smaller footprint
```

File: docs/guide/learning/how-to-learn-vue.md [hunk 12]
```diff
@@ -195,6 +195,34 @@ In this section, we will go over the most popular technologies and the struggles
...
+ - VueJs is less verbose
```
Comment (dobromir-hristov) [inline docs/guide/learning/how-to-learn-vue.md]:
```suggestion
 - Vue is less verbose - easier learning curve, lower entry point
```

File: docs/guide/learning/how-to-learn-vue.md [hunk 13]
```diff
@@ -195,6 +195,34 @@ In this section, we will go over the most popular technologies and the struggles
...
+ - VueJs community is active and evolving, while AngularJs one moved the attention to Angular (2x, 3x, 4x)
```
Comment (dobromir-hristov) [inline docs/guide/learning/how-to-learn-vue.md]:
```suggestion
 - The Vue community is active and evolving, while the AngularJs one shifted attention to Angular (2x, 3x, 4x) 
```

File: docs/guide/learning/how-to-learn-vue.md [hunk 14]
```diff
@@ -195,6 +195,34 @@ In this section, we will go over the most popular technologies and the struggles
...
+
+<useful-links>
+<useful-links-section title="Links">
```
Comment (dobromir-hristov) [inline docs/guide/learning/how-to-learn-vue.md]:
```suggestion
<useful-links-section title="Useful links"> 
```

File: docs/guide/learning/how-to-learn-vue.md (hunk 10 above)
Comment (DonNicoJs) [reply docs/guide/learning/how-to-learn-vue.md]: There is a typo, fixing it :)

**Comment by DonNicoJs** [comment]
@dobromir-hristov yap! All good minus a typo that I fixed
